css/templates/e-commerce/page.css

```diff
@@ -493,15 +522,29 @@ amp-selector [option][selected] {
   .commerce-product-thumb {
     width: 80px;
   }
-}
 
-@media (--breakpoint-md) {
   .commerce-product-desc {
     margin-right: calc(1/12 * 100%);
   }
 }
 
 
+/* BLOG */
+.commerce-blog-wrapper {
+  background-color: var(--color-white);
+}
+
+@media (--breakpoint-md) {
+  .commerce-blog {
+    margin-left: calc(1/12 * 100%);
+  }
+}
+
+.commerce-blog p {
+  line-height: 1.6;
```

Comment:
I don't think this should be a css var since it's only used once and shouldn't really be configured.